include/NAS2D/Common.h

```diff
@@ -45,8 +45,8 @@ std::pair<std::string, std::string> splitOnLast(const std::string& str, char del
 std::string join(std::vector<std::string> strs, char delim, bool skip_empty = true);
 std::string join(std::vector<std::string> strs, bool skip_empty = true);
 std::string trimWhitespace(std::string string);
-bool startsWith(const std::string& string, const std::string& start);
-bool endsWith(const std::string& string, const std::string& end);
+bool startsWith(const std::string& string, const std::string& start) noexcept;
+bool endsWith(const std::string& string, const std::string& end) noexcept;
```

Comment:
I suspect the reason for this would be good to add to a test case.

Reply:
See above about internal code quality tools.